### DISCUSSION TITLE
How do I change the wine version to use?

### DISCUSSION THREAD

**Discussion by devraza** [fufexan/nix-gaming#128] [Q&A] [answered]
Hi,

This is probably a pretty stupid question (new to Nix/NixOS here) - and I apologise for that (so I'll make this brief) - but I couldn't find a solution to this:

I'm running NixOS with Nixpkgs unstable here and osu!stable won't run with it's built-in wine version - after a little bit of testing, I've found that the wine-staging version 8.5 works.

What I want to ask, however, is how to override the osu!stable package to change the wine package used?
I've looked at the following snippet found on this project's `README.md`:
```nix
{
  environment.systemPackages = let
    gamePkgs = inputs.nix-gaming.packages.${pkgs.hostPlatform.system};
  in [ # or home.packages
    gamePkgs.osu-stable.override rec {
      wine = <your-wine>;
      wine-discord-ipc-bridge = gamePkgs.wine-discord-ipc-bridge.override {inherit wine;}; # or override this one as well
    };
  ];
}
```
And changed `wine` to `pkgs.winePackages.staging` but I don't think that's quite right as I get an error...

Help would truly be appreciated!

**Comment by fufexan** [comment]
That's the correct way to override. Can you post the error?

**Comment by NotAShelf** [comment, marked as answer]
you probably need to write the override in parenthesizes

like so:

```nix
{
  environment.systemPackages = let
    gamePkgs = inputs.nix-gaming.packages.${pkgs.hostPlatform.system};
  in [ # or home.packages
    (gamePkgs.osu-stable.override rec {
      wine = <your-wine>;
      wine-discord-ipc-bridge = gamePkgs.wine-discord-ipc-bridge.override {inherit wine;}; # or override this one as well
    });
  ];
}
```

**Reply by devraza** [reply]
Putting it in brackets fixed it pretty nicely - thank you very much!
